Client/Frontend/Widgets/PhotonActionSheet.swift

```diff
@@ -657,11 +657,27 @@ private class PhotonActionSheetCell: UITableViewCell {
         accessibilityLabel = action.title
         selectionStyle = action.handler != nil ? .default : .none
 
-        if let iconName = action.iconString, let image = UIImage(named: iconName)?.withRenderingMode(.alwaysTemplate) {
-            statusIcon.sd_setImage(with: action.iconURL, placeholderImage: image, options: []) { (img, err, _, _) in
-                if let img = img {
-                    self.statusIcon.image = img.createScaled(PhotonActionSheetUX.IconSize)
-                    self.statusIcon.layer.cornerRadius = PhotonActionSheetUX.IconSize.width / 2
+        let button = TabsButton().clone()
```

Comment:
Does `let button = TabsButton()` work, or is clone needed?

Reply:
The code following this to copy the UIView to a UIImage looks correct to me, it should work but I think you mentioned it doesn't.

Reply:
Both ways don't seem to give the expected result